.github/workflows/elastic-agent-project-board.yml

```diff
@@ -32,8 +32,8 @@ jobs:
         with:
           headers: '{"GraphQL-Features": "projects_next_graphql"}'
           query: |
-            mutation add_to_project($projectid:String!,$contentid:String!) {
-              updateIssue(input: {id:${{ github.event.issue.node_id }}, projectIds:$projectid}) {
+            mutation add_to_project($projectid:[ID!]!,$contentid:ID!) {
```

Comment:
I stumbled over `[ID!]!` but did not dig into how this exactly works in graphql. I'm sure you tested this one.

Reply:
I did! Project parameter can accept multiple values, that's why you need to add []